rpc/src/rpc.rs

```diff
@@ -1957,7 +1960,8 @@ impl JsonRpcRequestProcessor {
                                 }
                             })
                     },
-                    ScanConfig::default(),
+                    &ScanConfig::default(),
+                    bank.byte_limit_for_scans(),
```

Comment:
Should this be a field in `ScanConfig` ?

Reply:
I was imagining these were separate concepts. byte_limit doesn't go all the way to the scan itself. Seems more like client data.

Reply:
I see.  I suppose I saw it as a thing that effects the scan (and less plumbing).  Not a blocker either way